watertap/tools/unit_models/reverse_osmosis.py

```diff
@@ -0,0 +1,113 @@
+#################################################################################
+# WaterTAP Copyright (c) 2020-2024, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National Laboratory,
+# National Renewable Energy Laboratory, and National Energy Technology
+# Laboratory (subject to receipt of any required approvals from the U.S. Dept.
+# of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#################################################################################
+"""
+This module contains functions to be used with WaterTAP
+ReverseOsmosis0D or ReverseOsmosis1D unit models.
+"""
+
+from pyomo.environ import (
+    ConcreteModel,
+    assert_optimal_termination,
+    value,
+)
+from idaes.core.util.initialization import solve_indexed_blocks
+
+from watertap.property_models.seawater_prop_pack import SeawaterStateBlockData
+from watertap.property_models.NaCl_prop_pack import NaClStateBlockData
+from watertap.property_models.NaCl_T_dep_prop_pack import (
+    NaClStateBlockData as NaClTDepStateBlockData,
+)
+from watertap.core import MembraneChannel0DBlock, MembraneChannel1DBlock
+from watertap.core.solvers import get_solver
+
+
+__all__ = ["calculate_operating_pressure"]
+
+
+def calculate_operating_pressure(
+    feed_state_block=None,
+    over_pressure=0.15,
+    water_recovery_mass=0.5,
+    salt_passage=0.01,
```

Comment:
```suggestion
    salt_passage_fraction=0.01
```
Any reason not to have the default as 0? This is more in line with the typical rule of thumb calculation you'd make to estimate brine osmotic pressure.

Reply:
updated to 0